torch/onnx/utils.py

```diff
@@ -1073,7 +1073,8 @@ def _run_symbolic_function(g, block, n, inputs, env, operator_export_type=Operat
 
 # Generate an ONNX ATen op node.
 def _graph_at(g, opname, *args, **kwargs):
-    return g.op("ATen", *args, operator_s=opname, **kwargs)
+    overload_name = kwargs.pop('overload_name_s', '')
```

Comment:
IIUC, this function is currently doing two things:
1. Avoiding repetition of the string "ATen".
2. Ensuring `overload_name` defaults to empty string.

Is number 2 really needed? What happens if it's just absent?

If all we really need is 1, it seems like we could just replace the function with a constant like: `_ATEN_OPNAME`.

If we do need number 2, then I guess we should keep this function, but in that case I'd strongly suggest renaming it and some of the arguments to something like:
`def aten_op(g, operator, *args, overload_name="", **kwargs):`

Also note on line 1211 of this file we have:
`torch._C.Graph.at = _graph_at`

We should either delete that (my preference) or use it instead of calling this function directly.

Reply:
Refactored the helper to receive `overload_name` as function argument, which enforces default value and helps documenting the ATen op schema. Not having a schema for ATen nodes make future extension harder as users will not know about the overload attribute

`torch._C.Graph.at = _graph_at` enables `g.at()` idiom on custom ops, so it better stay. I will use `g.at` as opposed to `_graph_at` for consistency with existing `g.op` references.

`graph_at` was renamed to `_aten_op` for clarity